Sources/Observers/KeyboardObserver.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  KeyboardObserver.swift
+//  Alicerce
+//
+//  Created by Luís Portela on 17/05/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+public final class KeyboardObserver: NSObject {
+
+    fileprivate var keyboardVisible = false
```

Comment:
Would it make sense to provide delegate methods, like ```keyboardAppeared```and ```keyboardDisappeared``` to provide hooks to use in the client apps? I understand that we've got the NotificationCenter thingy, but whenever I've got to use it I kind of feel dirty... 🦆